### ISSUE TITLE
Call Method on Device does not send proper JSON

### ISSUE THREAD

**Issue by olavt** [Azure/azure-iot-sdk-csharp#73]
When using Device Explorer to call a method on a device proper JSON is not sent to the client device.

I entered this in the Device Explorer Method payload input box:

    {  "NodeId":2, "ParameterNumber":1, "ParameterValue":5, "ParameterSize":2 }

This is what's incorrectly being sent from Device Explorer to the client:

    "{  \"NodeId\":2, \"ParameterNumber\":1, \"ParameterValue\":5, \"ParameterSize\":2 }"

**Comment by tameraw** [comment]
@olavt - Can you also specify which version of DeviceExplorer you're using?

**Comment by olavt** [comment]
How do I find the version number? I downloaded it last week.

**Comment by sergaz-msft** [comment]
Hi @olavt , the latest version was released on the 10th of March: https://github.com/Azure/azure-iot-sdk-csharp/releases . Did you download the tool from there? You can also check the date modified from the DeviceExplorer.exe here C:\Program Files (x86)\Microsoft\DeviceExplorer. Thanks.

**Comment by sergaz-msft** [comment]
Hello again @olavt. I used the sample [DeviceClientMethodMqttSample](https://github.com/Azure/azure-iot-sdk-csharp/tree/master/device/samples/DeviceClientMethodMqttSample) and added 
```csharp 
dynamic parameters = JsonConvert.DeserializeObject(methodRequest.DataAsJson);
int nodeId = parameters.NodeId;
```

to the Method "WriteToConsole". When sending {  "NodeId":2, "ParameterNumber":1, "ParameterValue":5, "ParameterSize":2 } from DeviceExplorer (version 2017-3-10) I don't get any errors and the nodeId is correctly extracted from the payload.

Can you try the latest version of DeviceExplorer?

Thanks.

**Comment by olavt** [comment]
To download the Device Explorer I went to: https://github.com/Azure/azure-iot-sdk-csharp/tree/master/tools/DeviceExplorer then I clicked the link to get to the Download and got here: https://github.com/Azure/azure-iot-sdks/releases. The latest version there is 2016-11-17. I have now downloaded and tested the version from 2017-3-10 and it seems to work fine.

Why do we have two different places with releses of Device Explorer?
